debian/control

```diff
@@ -3,11 +3,11 @@ Maintainer: https://github.com/seznam/goenvtemplator
 Section: admin
 Priority: optional
 Build-Depends: debhelper (>= 9), golang (>= 2:1.5), git
-Replaces: goenvtemplator
-Breaks: goenvtemplator
 
 Package: goenvtemplator2
 Architecture: amd64
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Built-Using: ${misc:Built-Using}
```

Comment:
You don't need the `Built-Using:` here, see [Debian Policy](https://www.debian.org/doc/debian-policy/ch-relationships.html#s-built-using) or [Debian bug 688251](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=688251).